source/_integrations/probe_plus.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: Probe Plus
+description: Instructions on how to integrate Probe Plus food temperature probes into Home Assistant.
+ha_release: 2025.5.0
```

Comment:
2025.6

Reply:
fixed in https://github.com/home-assistant/home-assistant.io/pull/39110